internal/bminventory/inventory_test.go

```diff
@@ -7620,9 +7580,39 @@ var _ = Describe("DownloadMinimalInitrd", func() {
 		rootfsServiceConfig := fmt.Sprintf("[Service]\n"+
 			"Environment=http_proxy=%s\nEnvironment=https_proxy=%s\nEnvironment=no_proxy=%s\n"+
 			"Environment=HTTP_PROXY=%s\nEnvironment=HTTPS_PROXY=%s\nEnvironment=NO_PROXY=%s",
-			clusterProxyInfo.HTTPProxy, clusterProxyInfo.HTTPSProxy, clusterProxyInfo.NoProxy,
-			clusterProxyInfo.HTTPProxy, clusterProxyInfo.HTTPSProxy, clusterProxyInfo.NoProxy)
+			httpProxy, httpsProxy, noProxy,
+			httpProxy, httpsProxy, noProxy)
 		Expect(rootfsServiceConfigContent).To(Equal(rootfsServiceConfig))
+	}
+
+	It("returns not found with a non-existant cluster", func() {
+		infraEnv := createInfraEnv(models.ImageTypeFullIso)
+		storeInfraEnv(infraEnv)
+		params := installer.DownloadMinimalInitrdParams{InfraEnvID: strfmt.UUID(uuid.New().String())}
+		response := bm.DownloadMinimalInitrd(ctx, params)
+		verifyApiError(response, http.StatusNotFound)
+	})
+
+	It("returns no content without network customizations", func() {
+		infraEnv := createInfraEnv(models.ImageTypeFullIso)
+		storeInfraEnv(infraEnv)
+		params := installer.DownloadMinimalInitrdParams{InfraEnvID: id}
+		response := bm.DownloadMinimalInitrd(ctx, params)
+		Expect(response).Should(BeAssignableToTypeOf(&installer.DownloadMinimalInitrdNoContent{}))
+	})
+
+	It("returns legit archive when full ISO is requested", func() {
```

Comment:
I think I asked about this before, but while we're here can we clarify what "legit" is?